specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/stable/2022-01-01/nodetype.json

```diff
@@ -0,0 +1,1488 @@
+          },
+          {
+            "value": "Premium_LRS",
+            "description": "Premium SSD locally redundant storage. Best for production and performance sensitive workloads."
+          }
+        ]
+      }
+    },
+    "FrontendConfiguration": {
+      "type": "object",
+      "properties": {
+        "ipAddressType": {
+          "$ref": "#/definitions/IPAddressType",
+          "description": "The IP address type of this frontend configuration. If omitted the default value is IPv4."
+        },
+        "loadBalancerBackendAddressPoolId": {
+          "type": "string",
+          "description": "The resource Id of the Load Balancer backend address pool that the VM instances of the node type are associated with. The format of the resource Id is '/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/loadBalancers/<loadBalancerName>/backendAddressPools/<backendAddressPoolName>'."
+        },
+        "loadBalancerInboundNatPoolId": {
+          "type": "string",
+          "description": "The resource Id of the Load Balancer inbound NAT pool that the VM instances of the node type are associated with. The format of the resource Id is '/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/loadBalancers/<loadBalancerName>/inboundNatPools/<inboundNatPoolName>'."
+        }
+      },
+      "description": "Describes the frontend configurations for the node type."
+    },
+    "IPAddressType": {
+      "type": "string",
+      "description": "The IP address type.\n",
+      "enum": [
+        "IPv4",
+        "IPv6"
+      ],
+      "default": "IPv4",
+      "x-ms-enum": {
+        "name": "IPAddressType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "IPv4",
+            "description": "IPv4 address type."
+          },
+          {
+            "value": "IPv6",
+            "description": "IPv6 address type."
+          }
+        ]
+      }
+    },
+    "ManagedProxyResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Azure resource identifier.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Azure resource name.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Azure resource type.",
+          "readOnly": true
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Azure resource tags.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        },
+        "location": {
+          "type": "string",
```

Comment:
Proxy resource should not have a location
Please refer 'Comparison of Tracked Resources and proxy only Resources' in [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#designing-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#designing-resources)